packages/replay-next/components/sources/useSourceContextMenu.tsx

```diff
@@ -144,7 +144,11 @@ function FastForwardButton({
 
   const fastForward = () => {
     assert(fastForwardToExecutionPoint != null);
-    update(fastForwardToExecutionPoint.time, fastForwardToExecutionPoint.point, false);
+    update(fastForwardToExecutionPoint.time, fastForwardToExecutionPoint.point, false, {
+      sourceId,
+      line: lineNumber,
+      column: lineHitCounts?.firstBreakableColumnIndex ?? 0,
```

Comment:
How does this fallback (`?? 0`) relate to FE-2015?

Reply:
We're not treating this location as a breakable location (so we won't request hit points for it). But actually this method should never be called if `lineHitCounts` is `null`, so I'll replace the fallback with an assert to make that clear (there's already `assert(fastForwardToExecutionPoint != null);` in there and `fastForwardToExecutionPoint` would be `null` if `lineHitCounts` was)